feature-libs/asm/core/store/actions/asm-ui.action.ts

```diff
@@ -4,7 +4,7 @@ import { AsmUi } from '../../models/asm.models';
 export const ASM_UI_UPDATE = '[Asm] UI Update';
 
 export class AsmUiUpdate implements Action {
-  readonly type = ASM_UI_UPDATE;
+  readonly type: string = ASM_UI_UPDATE;
```

Comment:
Was adding the `string` required here?

Reply:
Fixed.  Good catch.  You're right.  It's not needed.
I think it must have been a side effect of a solution I tried somewhere else and then reverted because I remember the compilation error that led me there.  Go figure.